monitoring/README.rst

```diff
@@ -1,78 +1,100 @@
-Python Client for Stackdriver Monitoring
-========================================
+Python Client for Stackdriver Monitoring API (`Alpha`_)
+=======================================================
 
-    Python idiomatic client for `Stackdriver Monitoring`_
+`Stackdriver Monitoring API`_: Manages your Stackdriver Monitoring data and configurations. Most projects must be associated with a Stackdriver account, with a few exceptions as noted on the individual method pages.
 
-.. _Stackdriver Monitoring: https://cloud.google.com/monitoring/
 
-|pypi| |versions|
+- `Client Library Documentation`_
+- `Product Documentation`_
 
--  `Documentation`_
-
-.. _Documentation: https://googlecloudplatform.github.io/google-cloud-python/latest/monitoring/usage.html
+.. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
+.. _Stackdriver Monitoring API: https://cloud.google.com/monitoring
+.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/monitoring-usage
```

Comment:
This link is broken. Replace this with the following?
https://googlecloudplatform.github.io/google-cloud-python/latest/monitoring/usage.html